### PR TITLE
Adding HPE MPI (+MPT) package and external package configuration

### PR DESCRIPTION
**Pull request by pramodk**

### CODE REVIEW

File: sysconfig/bb5/tools/modules.yaml [hunk 1]
```diff
@@ -102,6 +103,7 @@ modules:
       - mvapich2
       - mpich
       - intel-mpi
+      - hpe-mpi
```
Comment (pramodk) [inline sysconfig/bb5/tools/modules.yaml:106]: yaml config is duplicated for tcl and lmod. will fix this as separate PR.

**Comment by timocafe** [comment]
Hello,

Does the knl works with hpe mpi ? I understand it was not stable ?

++t

Envoyé de mon GameBoy

**Comment by pramodk** [comment]
> Does the knl works with hpe mpi ? I understand it was not stable ?

yes, since last 8 hours :) just forwarded you the email.

**Review by ohm314 — APPROVED** [review]
LGTM

**Pull request merged**